ui/panels-plugin/src/plugins/line-chart/tooltip/Tooltip.tsx

```diff
@@ -13,7 +13,7 @@
 
 import { Box } from '@mui/material';
 import { useEffect, useState } from 'react';
-import { Coordinate, TooltipData, TOOLTIP_MIN_WIDTH } from './tooltip-model';
+import { Coordinate, TooltipData, TOOLTIP_MAX_HEIGHT, TOOLTIP_MAX_WIDTH } from './tooltip-model';
 import TooltipContent from './TooltipContent';
 
 function assembleTransform(coords: Coordinate, chartWidth: number) {
```

Comment:
Since TOOLTIP_MAX_WIDTH has increased, how `flipTooltipPosThreshold` is calculated may need to be adjusted so the tooltip doesn't get cut off by surrounding components like OptionsDrawer. Definitely not a blocker for this PR, I can take care of it when I look into using Portals so it attaches to the body (see [Luke comment](https://github.com/perses/perses/pull/266#discussion_r781511724) for reference)

Reply:
I agree, there needs to be some adjustments here, I wonder if it's some combination of tooltip container size, mouse position and browser window width  & height ?

Reply:
Yeah that's true, so many factors!